docs/src/components/InfiniteHits.md

```diff
@@ -0,0 +1,37 @@
+---
+title: InfiniteHits
+mainTitle: Components
+layout: main.pug
+category: Components
+withHeadings: true
+navWeight: 6
+editable: true
+githubSource: docs/src/components/InfiniteHits.md
+---
+
+Displays an infinite list of hits along with a load more button.
+
+<a class="btn btn-static-theme" href="stories/?selectedKind=InfiniteHits">🕹 try out live</a>
+
+## Usage
+
+```html
+<ais-infinite-hits></ais-infinite-hits>
+```
+
```

Comment:
Yes but we can add it later as a global guide cross InstantSearch, WDYT? Because most of the users will use a framework specific component (like [`vue-infinite-scroll`](https://github.com/ElemeFE/vue-infinite-scroll) for example).